conf/defaults.ini

```diff
@@ -165,6 +165,9 @@ connstr =
 # prefix prepended to all the keys in the remote cache
 prefix =
 
+# This enables encryption of values stored in the remote cache
```

Comment:
we need to add this into [the docs](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana)